scanner/src/main/kotlin/ScannerFactory.kt

```diff
@@ -47,7 +47,7 @@ abstract class AbstractScannerFactory<out T : Scanner>(
     abstract override fun create(config: ScannerConfiguration): T
 
     /**
-     * Return the scanner's name here to allow JCommander to display something meaningful when listing the scanners
+     * Return the scanner's name here to allow Clikt to display something meaningful when listing the scanners
```

Comment:
Commit message: Here, no common file / directory scope applies, so I'd just say "Fix outdated references to JCommander in Factory.toString()".